index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+    <link rel="stylesheet" type="text/css" href="style.css"/>
+</head>
+<body>
+    <title>
+    Ariel's Portfolio
+    </title>
+    <section id="one">
+        <img class= "logo alignment" src="avatar.jpg">
+        <nav class="width">
+            <ul>
+                <li><a href="#one">Home</a></li>
+                <li><a href="#two">About Me</a></li>
+                <li><a href="#three">Portfolio</a></li>
+                <li><a href="#four">Contact</a></li>
+            </ul>
+            <div class="hamburger"></div>
+            <div class="hamburger"></div>
+            <div class="hamburger"></div>
+        </nav>
+    </section>
+
+    <section id="two" class="width alignment">
```

Comment:
section id name needs to be more descriptive eg: aboutContent

Reply:
Done